packages/js-sdk/src/contract-helpers.ts

```
@@ -0,0 +1,114 @@
+    }
+    const headers = {
+        'Content-type': 'application/json',
+        Authorization: 'Bearer ' + accessToken,
+    };
+
+    try {
+        const response: AxiosResponse = await keypClient.post('/contracts/method/read', {
+            address: address,
+            abi: abi,
+            args: args,
+        }, {
+            headers,
+        });
+
+        return {
+            status: response.data.status,
+            explorerUrl: response.data.explorerUrl,
+            response: response.data.response,
+        };
+    } catch (error) {
+        console.log(error.message, 'error')
+        return { status: 'FAILURE', explorerUrl: '', response: { type: '', hex: ''} };
+    }
+};
+
+/**
+ * Writes to a contract method
+ * @param {string} accessToken - The access token for authentication
+ * @param {string} address - The address of the contract
+ * @param {string} abi - The ABI of the contract
+ * @param {string[]} args - The arguments for the contract method
+ * @param {string} value - (Optional) Amount of network coin to send in units of wei
+ */
+const writeContract = async ({accessToken, address, abi, args, value}: WriteContractParams): Promise<WriteContractResult> => {
+    if (!accessToken) {
+        throw 'No access token provided';
+    }
+    const headers = {
+        'Content-type': 'application/json',
+        Authorization: 'Bearer ' + accessToken,
+    };
+
+    try {
+        const response: AxiosResponse = await keypClient.post('/contracts/method/write', {
+            address: address,
+            abi: abi,
+            args: args,
+            value: value,
+        }, {
+            headers,
+        });
+
+        return {
+            status: response.data.status,
+            hash: response.data.hash,
+            explorerUrl: response.data.explorerUrl,
+            tx: response.data.tx,
+        };
+    } catch (error) {
+        return { status: 'FAILURE', hash: '', explorerUrl: '', tx: { type: '', chainId: ''} };
```

Did you add this try/catch to alleviate the missing error response from the API? If so, lets take these out, so we get the real response from the API. And we can make an issue to fix the API errors instead

---

removed the try catch to avoid eating the error. will make a separate issue for API errors